app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceType.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+
+class AddPoliceType : OsmFilterQuestType<PoliceType>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          amenity = police
+          and !operator
+    """
+    override val commitMessage = "Add police type"
+    override val wikiLink = "Tag:amenity=police"
+    override val icon = R.drawable.ic_quest_police
+    override val enabledInCountries = NoCountriesExcept(
+        "IT"
+    )
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_policeType_title
+
+    override fun createForm() = AddPoliceTypeForm()
+
+    override fun applyAnswerTo(answer: PoliceType, changes: StringMapChangesBuilder) {
+        changes.add("name", answer.policeName);
+        changes.add("operator", answer.operator.operatorName);
+        changes.add("operator:wikidata", answer.operator.wikidata);
```

Comment:
⚠️ 
This will result in an error if `operator:wikidata` is already set. The same for the other keys.
If you want to explicitly replace the value if it is already set, use `changes.addOrModify`

Reply:
It was decided to just not add generic names, so I'll remove it from this quest. Maybe we could add police to the name quest? I don't know if it's there already but maybe it could be useful. 🤔